### PR TITLE
Show example of multiple UseStaticFiles

### LINKED ISSUE #24559: Just use UseStaticFiles twice
This popular SO question shows that people are confused about how to serve extra file locations as well as wwwroot: https://stackoverflow.com/questions/41021715/how-can-you-use-multiple-directories-for-static-files-in-an-aspnet-core-app/41021991#41021991

The answer to serving both wwwroot and another directory is to use static files twice:

```
app.UseStaticFiles();
app.UseStaticFiles(new StaticFileOptions
{
    FileProvider = new PhysicalFileProvider(
        Path.Combine(Directory.GetCurrentDirectory(), "static"))
});
```
However in the section **Serve files outside of web root** it only uses a single UseStaticFiles but the text gives the impression that wwwroot will also be server, when in fact the code does not do this.

Suggestion:

Update the text to say that if you want to serve other directories as well as wwwroot, you need to have a parameterless UseStaticFiles plus extra UseStaticFiles for each other directory. Add two UseStaticFiles to the code

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d47f38e2-bc36-cfb8-b0cc-17a534689f8b
* Version Independent ID: 3fec6e08-fc99-7a5c-796f-3f2347cad891
* Content: [Static files in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/static-files?view=aspnetcore-6.0&viewFallbackFrom=aspnetcore-3.0#serve-files-outside-of-web-root)
* Content Source: [aspnetcore/fundamentals/static-files.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/fundamentals/static-files.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by serpent5**
Fixes #24559.

[Internal Preview URL](https://review.docs.microsoft.com/en-us/aspnet/core/fundamentals/static-files?view=aspnetcore-6.0&branch=pr-en-us-24576#serve-files-from-multiple-locations).

### CODE REVIEW

**Comment by serpent5** [comment]
@Rick-Anderson Once we get approval on this, I'd like to fix the markdown issues and convert the snippet reference syntax before merging.

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
> @Rick-Anderson Once we get approval on this, I'd like to fix the markdown issues and convert the snippet reference syntax before merging.

Can you do that in another PR?

**Comment by Rick-Anderson** [comment]
> @Rick-Anderson Once we get approval on this, I'd like to fix the markdown issues and convert the snippet reference syntax before merging.

Can you do that in another PR? I'd like to get this out today.

**Pull request merged**
